ConsumerApi/Mvc/Middleware/UserDataLoggingMiddleware.cs

```diff
@@ -0,0 +1,43 @@
+﻿using Enmeshed.BuildingBlocks.Application.Abstractions.Infrastructure.UserContext;
+using Serilog.Context;
+using Serilog.Core;
+using Serilog.Core.Enrichers;
+
+namespace ConsumerApi.Mvc.Middleware;
+
+public class UserDataLoggingMiddleware
+{
+    private readonly RequestDelegate _next;
+    private readonly IUserContext _userContext;
+
+    public UserDataLoggingMiddleware(RequestDelegate next, IUserContext userContext)
+    {
+        _next = next;
+        _userContext = userContext;
+    }
+
+    public async Task Invoke(HttpContext context)
+    {
+        try
+        {
+            var deviceId = _userContext.GetDeviceId();
+            var identityAddress = _userContext.GetAddress();
+
+            ILogEventEnricher[] enrichers =
```

Comment:
Isn't it possible to use `var` here?

Reply:
It is if we do the following:

```c#
var enrichers = new ILogEventEnricher[]
{
    new PropertyEnricher("deviceId", deviceId),
    new PropertyEnricher("identityAddress", identityAddress)
};
```

Is that preferred?